src/main/msp/msp_protocol.h

```diff
@@ -62,7 +62,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   41 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   42 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
This should not be in here.

Reply:
yes I understand that, however the comments in the MSP code won't match the code.

I targeted this for 4.1, so when 4.0 is done the API version should be bumped to 42 by another commit and this change will disappear right?  No problem removing for you now if you like though.

Reply:
Yes please, remove. Otherwise we'll get back to the bad old days where every pull request including an MSP change will increment this 'because it's needed', leading to an increment of 10 or so between releases...

Reply:
done